onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -510,11 +510,10 @@ def main():
     if args.input_int32:
         optimizer.change_graph_inputs_to_int32()
 
-    if args.model_type in set(MODEL_TYPES.keys()):
-        if optimizer.is_fully_optimized():
-            logger.info("The model has been fully optimized.")
-        else:
-            logger.info("The model has been optimized.")
+    if "bert" in args.model_type and optimizer.is_fully_optimized():
+        logger.info("The model has been fully optimized.")
```

Comment:
Why is `bert` the only model checked vs keeping the current code to check all model types?

Reply:
is_fully_optimized is not implemented by other model type like unet etc.